esmvaltool/diag_scripts/shared/scaling.ncl

```diff
@@ -266,7 +266,12 @@ begin
     end if
   end if
 
+  if (units_from.eq.units_to) then
```

Comment:
I would put this block at the very beginning, to save parsing all the if statements for nothing.

Reply:
Makes sense!